com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -520,6 +520,17 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             if (useRenderPassEnabled || useDepthPriming)
                 createColorTexture |= createDepthTexture;
 
+            //Set rt descriptors so preview camera's have access should it be needed
+            CommandBuffer cmd = CommandBufferPool.Get();
+            var colorDescriptor = cameraTargetDescriptor;
+            colorDescriptor.useMipMap = false;
+            colorDescriptor.autoGenerateMips = false;
+            colorDescriptor.depthBufferBits = (int)DepthBits.None;
```

Comment:
Should this be moved to the place we create the texture descriptor? https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs?L574

Reply:
The descriptor is used for many other RT's during the pipeline where I don't see these values being changed so I am not certain we want to apply this everywhere. DepthPrepass, normals buffer, motion buffer, ...

I think it's safer to keep it this way